setup.py

```diff
@@ -17,19 +17,15 @@
         "click",
         "flask",
         "mailer",
+        "pre-commit",
```

Comment:
Decided to just move the dependencies from the "testing" extras to here, because in practice we always install all dependencies in the docker image in order to run tests in it.